tests/chart_tests/test_astronomer_config_syncer.py

```diff
@@ -278,3 +278,23 @@ def test_astronomer_config_syncer_disabled(self, kube_version):
         )
 
         assert len(docs) == 0
+
+    def test_astronomer_config_syncer_and_global_rbac_enabled_with_sa_create_disabled(self, kube_version):
+        """Test that config syncer service account, rbac  is disabled when config syncer and global rbac is enabled."""
```

Comment:
This wording seems better, assuming I understood the original comment correctly.
```suggestion
        """Test that config syncer service account and rbac are disabled when config syncer and global rbac is enabled."""
```